assets/model_monitoring/components/src/model_data_collector_preprocessor/store_url.py

```diff
@@ -68,7 +68,7 @@ def get_default_credential():
             elif self._datastore.credential_type == "Sas":
                 return AzureSasCredential(self._datastore.sas_token)
             elif self._datastore.credential_type is None:
-                return get_default_credential()  # credential less datastore
+                raise InvalidInputError("Credential-less input data is NOT supported for Model Monitoring job.")
```

Comment:
could you please print out the datastore name and ask customer to change to credential datastore

Reply:
addressed.